tests/semantic_analyser.cpp

```diff
@@ -3253,6 +3253,27 @@ TEST(semantic_analyser, strconrtains)
   test("i:s:1 { strcontains(\"a\",1) }", 2);
 }
 
+TEST(semantic_analyser, strcontains_large_warnings)
+{
+  test_for_warning("kprobe:f { $s1 = "
+                   "str(arg0); $s2 = str(arg1); strcontains($s1, $s2) }",
```

Comment:
A bit hard to read, improve line splits?
```suggestion
  test_for_warning("kprobe:f {
                   "$s1 = str(arg0); $s2 = str(arg1);
                   "strcontains($s1, $s2) }",
```